10-inference.Rmd

```diff
@@ -83,124 +87,120 @@ The population consists of all studio apartment rentals in Vancouver, and the po
-and chocolate&mdash;as our population, and use this to illustrate
-inferential concepts. Below we have a `tibble()` called `virtual_box` with a Timbit ID and flavour as our columns. We have also loaded our necessary packages: `tidyverse` and the `infer` package, which we will need to perform sampling later in the chapter.  
-
-```{r 11-example-proportions, echo = FALSE, message = FALSE, warning = FALSE}
-library(gridExtra)
-set.seed(1234)
-virtual_box <- tibble(
-  timbit_id = seq(1, 10000, by = 1),
-  flavour = factor(rbinom(10000, 1, 0.63),
-    labels = c("old fashioned", "chocolate")
-  )
-)
+```{r}
+airbnb
 ```
+Suppose the city of Vancouver wants to know how many Airbnb places are listed as entire homes and apartments (rather than as private or shared rooms). Therefore they may want to estimate the true proportion of all Airbnb listings where the "type of place" is listed as "entire home or apartment." Of course, we usually do not have access to the true population, but here let's imagine (for learning purposes) that our data set represents the population of all Airbnb rental listings in Vancouver, Canada. We can find the proportion of listings where `room_type == "Entire home/apt"`.
 
-```{r 11-example-proportions1}
+```{r 11-example-proportions2, echo = TRUE, message = FALSE, warning = FALSE }
 library(tidyverse)
 library(infer)
-virtual_box
-```
-
-From our simulated box, we can see that the proportion of chocolate Timbits is
-0.63. This value, 0.63, is the *population parameter*. Note that this parameter value is usually unknown in real data analysis problems.
 
-```{r 11-example-proportions2, echo = TRUE, message = FALSE, warning = FALSE}
-virtual_box %>%
-  group_by(flavour) %>%
+airbnb %>%
   summarize(
-    n = n(),
-    proportion = n() / 10000
+    n =  sum(room_type == "Entire home/apt"),
+    proportion = sum(room_type == "Entire home/apt") / nrow(airbnb)
   )
 ```
 
-What would happen if we were to buy a box of 40 randomly-selected Timbits and count the number of chocolate Timbits (*i.e.,* take a random sample of size 40 from our Timbits population)? Let's use R to simulate this using our `virtual_box` population. We can do this using the `rep_sample_n` function from the `infer` package. The arguments
-of `rep_sample_n` are (1) the data frame (or tibble) to sample from, and (2) the size of the sample to take.
+```{r 11-population-parameter, echo = F}
+population_proportion <- airbnb %>%
+  summarize(
+    n =  sum(room_type == "Entire home/apt"),
+    proportion = sum(room_type == "Entire home/apt") / nrow(airbnb)
+  ) %>% 
+  pull()
+```
+
+From our data, we can see that the proportion of `Entire home/apt` listings in the data set is `r round(population_proportion,3)`. This value, `r round(population_proportion,3)`, is the *population parameter*. Remember, this parameter value is usually unknown in real data analysis problems.
```

Comment:
inconsistency here whether we emphasize important terms with italics or boldface. i think the rest of the chapter uses bold